keras/ops/numpy_test.py

```diff
@@ -3880,6 +3880,9 @@ def test_arange(self):
         self.assertAllClose(knp.Arange()(3, 7, 2), np.arange(3, 7, 2))
 
         self.assertEqual(standardize_dtype(knp.arange(3).dtype), "int32")
+        with pytest.warns(None) as record:
+            knp.arange(3, dtype="int")
```

Comment:
This is breaking with TF because `"int"` is not being converted to an int dtype somehow.